workspace/utilities/lib/util-nav-link-attr.xsl

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: util-nav-link-attr -->
+	<xsl:template name="util-nav-link-attr">
+		<xsl:param name="label" select="label" />
+		<xsl:param name="ga-cat" select="'menu-nav'" />
+		<xsl:param name="ga-label" select="concat($url-language, '/', $label)" />
+		<xsl:param name="role" select="'menuitem'" />
+		<xsl:param name="target" select="'_self'" />
+		<xsl:param name="ext-attr" />
```

Comment:
not necessary

Reply:
But can it be someday? Other than that it's just cleaner to have all values up top don't you think?

Reply:
OOOOOOhhhh, nevermind, didn't understood you were talking about **ext-attr** only.